app/_data/kong_versions.yml

```diff
@@ -385,13 +385,14 @@
   endOfLifeDate: "2024-11-15"
   branch: release-2.5
 - edition: mesh
-  version: 2.6.0
+  version: 2.6.1
   release: 2.6.x
   latest: true
   releaseDate: "2024-02-01"
   endOfLifeDate: "2025-02-01"
   branch: release-2.6
 - edition: mesh
   version: preview
```

Comment:
i think this `version: preview` might still be breaking the output: https://deploy-preview-6973--kongdocs.netlify.app/mesh/dev/

Reply:
@lahabana is looking into that

Reply:
Hmm wondering if I have maybe switched the 2 args around :(